plugins/module_utils/vcenter_utils.py

```diff
@@ -15,96 +15,116 @@
 import requests
 import atexit
 
-from pyVim import connect
+from pyvim import connect
 from pyVmomi import vmodl
 from pyVmomi import vim
 
-def establish_vcenter_connection(module, vCenter_host, username, password):
-    '''
+
+def establish_vcenter_connection(module, vCenter_host, username, password, ignore_ssl_verification):
+    """
     params:
     - vCenter_host: vCenter host IP
     - username: vCenter username
     - password: vCenter password
     result:
-    Retrieves vCenter information from service instance and returns as content object. 
-    '''
-    try:
-        service_instance = connect.SmartConnect(host=vCenter_host,
-                                                user=username,
-                                                pwd=password,
-                                                port=443)
-        if not service_instance:
-            module.fail_json(msg="Could not connect to the specified vCenter "
-                  "host using specified username and password")
-
-        atexit.register(connect.Disconnect, service_instance)
-    except (requests.ConnectionError, ssl.SSLError):
+    Retrieves vCenter information from service instance and returns as content object.
```

Comment:
Bug Fixed - 2731783 - Getting "info from vCenter: [SSL: CERTIFICATE_VERIFY_FAILED]" when deploying 2nd and 3rd NSX-T managers

Logic - 
Added a new flag ignore_ssl_connection to decide whether to use SSL while connecting to VC or not.